nautobot/core/templates/generic/object_list.html

```diff
@@ -117,6 +118,18 @@ <h1>{% block title %}{{ title }}{% endblock %}</h1>
                                 </label>
                             </div>
                             <div class="pull-right">
+                                {% if content_type.model_class.is_static_group_associable_model and perms.extras.add_staticgroupassociation %}
+                                    <button type="button"
+                                            id="update_static_groups_for_all"
+                                            data-toggle="modal"
+                                            data-target="#static_group_assignment_modal"
+                                            data-objects="all"
+                                            class="btn btn-primary btn-sm"
+                                            disabled="disabled">
+                                        <span class="mdi mdi-group" aria-hidden="true"></span>
+                                        Update Static Groups for All
+                                    </button>
+                                {% endif %}
```

Comment:
Sorry, need what?

Reply:
Sorry should made myself clearer. I am wondering if we should have what you did here for the `Add Components` button on the DeviceListView, something like `Add Components for All`
![Screenshot 2024-05-06 at 3 15 08 PM](https://github.com/nautobot/nautobot/assets/46973263/f6dc49f1-3e8d-499e-a721-98ff4f0d5c54)

Reply:
Ah gotcha. It'd be nice if we could come up with a way for this behavior to be automatic rather than needing to declare two different buttons for each bulk action.